docs/how-it-works.mdx

```diff
@@ -0,0 +1,116 @@
+---
+sidebar_position: 2
+---
+
+# How it Works
+
+Learn the high-level workings of Divvi Up and how it accomplishes the
+privacy-preserving property.
+
+## Breakdown
+
+Divvi Up functionality can be broadly broken down into five steps.
+
+See the [Distributed Aggregation Protocol (DAP)][DAP] for a highly detailed
+specification of the protocol that underpins Divvi Up.
+
+<!-- TODO(inahga): Consider making this a more detailed `mermaid` diagram -->
+
+![](../static/how-it-works.png#gh-dark-mode-only)
+![](../static/how-it-works-light.jpg#gh-light-mode-only)
+
+### Step 1: Client generate measurements
+
+An app developer wants to collect statistics about their app, such as user
+behavior, demographics, or system information. The developer writes app
+functionality that takes measurements of these properties.
+
+In Divvi Up, the developer creates a Task, which encodes data about the
+measurement being taken. This encoded data is stored in the app and used by
+client libraries. The data includes the measurement type, the two Divvi Up
+aggregators being used, the identifier for the task, and how aggregates are to
+be queried. The measurement type includes, but is not limited to, counts, sums,
+and histograms.
+
+For each measurement, the subsequent steps are performed.
+
+### Step 2: Client divides the measurement
+
+Using a client library, the app divides the measurement into two shares, known
+as report shares. One report share is meant for a Leader aggregator, and a
+Helper aggregator. Each share is encrypted with the respective aggregator's
+public key.
```

Comment:
As a naïve reader: "Where do I get this key?"

It's probably worth saying the DAP client knows where to get the key from the internet and will do so automatically